num_scouts: 2
